frontend/src/store/course/index.js

```diff
@@ -0,0 +1,79 @@
+import { questionMapper } from '@/utils/api/questions'
+import { reviewMapper } from '@/utils/api/reviews'
+import { courseMapper } from '@/utils/api/course'
+
+import { doRequestFactory } from '@/store/utils'
+
+import { REQUEST, COMMITS, ACTIONS } from './constants'
+
+const state = {
+  loading: false,
+  courseTab: 'info',
+  questions: [],
+  reviews: [],
+  course: {
+    id: 0,
+    code: '',
+    name: '',
+    rating: 0
+  },
+  error: {
+    code: 0,
+    message: ''
+  }
+}
+
+const getters = {
```

Comment:
If you're not doing anything special with the getters and you're using mappers in the component you don't really need getters.

Reply:
true but ceebs to change it now lel